src/push.rs

```diff
@@ -169,7 +167,9 @@ pub async fn build_profile_remotely(data: &PushProfileData<'_>, derivation_name:
 
 
     // copy the derivation to remote host so it can be built there
-    let copy_command_status = Command::new("nix").arg("copy")
+    let copy_command_status = Command::new("nix")
+        .arg("--experimental-features").arg("nix-command")
```

Comment:
I wonder if it makes sense to add `--experimental-features=nix-command` only if `!data.supports_flakes`, what do you think?